urbit/lib/eth-watcher.hoon

```diff
@@ -0,0 +1,60 @@
+/-  eth-watcher
+=,  ethereum-types
+|%
+::
+++  event-log-json
```

Comment:
Good to have json conversions for eth-watcher around! Best practice to wrap these in a `++ enjs  |%` core within the outer library core, for namespacing reasons. (Then you get to remove the `-json` suffixes etc, too.)

Reply:
:+1: